skll/learner.py

```diff
@@ -44,20 +44,29 @@
 from sklearn.feature_extraction import FeatureHasher
 from sklearn.feature_selection import SelectKBest
 # AdditiveChi2Sampler is used indirectly, so ignore linting message
-from sklearn.kernel_approximation import (AdditiveChi2Sampler,
-                                          Nystroem,
+from sklearn.kernel_approximation import (Nystroem,
                                           RBFSampler,
                                           SkewedChi2Sampler)
-from sklearn.linear_model import (ElasticNet, Lasso, LinearRegression,
-                                  LogisticRegression, Ridge, SGDClassifier,
-                                  SGDRegressor)
+from sklearn.linear_model import (BayesianRidge,
+                                  ElasticNet,
+                                  HuberRegressor,
+                                  Lars,
+                                  Lasso,
+                                  LinearRegression,
+                                  LogisticRegression,
+                                  RANSACRegressor,
+                                  Ridge,
+                                  RidgeClassifier,
+                                  SGDClassifier,
+                                  SGDRegressor,
+                                  TheilSenRegressor)
 from sklearn.linear_model.base import LinearModel
 from sklearn.metrics import (accuracy_score,
                              confusion_matrix,
-                             precision_recall_fscore_support,
-                             SCORERS)
```

Comment:
do we not need this any more?

Reply:
It was imported but never used in that file so I removed it.

Reply:
Ah, makes sense, thanks!